frontend/src/app/core/helpers/language-helper.ts

```diff
@@ -0,0 +1,36 @@
+import { Language } from '@shared/enums/language';
+
+export const getLanguageEnum = (userLanguageBrowser: string) => {
+    let userLanguage: Language = Language.Eng;
+
+    const languages: { enum: Language, text: string } [] = [
+        {
+            text: 'en',
+            enum: Language.Eng,
+        },
+        {
+            text: 'uk',
+            enum: Language.Ukr,
+        },
+        {
+            text: 'pl',
+            enum: Language.Pl,
+        },
+        {
+            text: 'fr',
+            enum: Language.Fr,
+        },
+        {
+            text: 'it',
+            enum: Language.It,
+        },
+    ];
+
+    languages.forEach(lang => {
+        if (userLanguageBrowser.includes(lang.text)) {
+            userLanguage = lang.enum;
+        }
+    });
```

Comment:
what is this part supposed to do?

Reply:
It will iterate through array with pairs of languages enums and texts. If language of the user contains part of the text, the corresponding enum will be selected. We can have "en" or "en-us" from browser api and these two are english languages, so I look for substrings of languages

Reply:
Refactored helper)